services/proton-rpc.ts

```diff
@@ -57,6 +61,41 @@ class ProtonJs {
     const { avatar } = rows[0];
     return avatar;
   };
+
+  getAtomicMarketBalance = (chainAccount: string) => {
```

Comment:
When you rebase, this will be deleted, but you need to keep this function here because we need to pull this balance for withdrawal amounts for the "claim $$ from sales" banner